### PR TITLE
Updated Navbar styling/Include Dropdown menu for Match Lobby

### PR DESCRIPTION
**Pull request by leekahung**
Have gone over styling for Navbar components and included Dropdown menus for Match Lobby to show Current Games and Completed Games. Have moved all Navbar related components into its own directory called Navbar.

New Dropdown menus uses HTML ids for internal routing (see below). Current Games and Completed Games are /match#current-games and /match#completed-games, respectively. I think there's a better way to route these in Next.js, but I'm not sure how. Feedback would be great!

https://user-images.githubusercontent.com/14917816/235410648-02105507-a9a6-4323-bf76-2e860692ece6.mov

Had accounted for when Navbar becomes burger menu below 768px width.

https://user-images.githubusercontent.com/14917816/235410708-e8d18535-2a16-469f-a345-179212ae4a42.mov

### CODE REVIEW

**Comment by leekahung** [comment]
What is the current status of PR #96 at the moment?

File: src/pages/match/index.tsx [hunk 1]
```diff
@@ -65,7 +64,7 @@ export default function Games() {
                 );
               })}
         </div>
-        <div className="completedGames">
+        <div id="completed-games" className="completedGames">
```
Comment (FemboyDeveloper) [inline src/pages/match/index.tsx:67]: Why is the id in kebab-case but the class in camelCase? Please make sure your casing is consistent.

File: src/pages/match/index.tsx [hunk 2]
```diff
@@ -34,12 +34,11 @@ const MatchRow = ({ match }: { match: Match }) => {
 export default function Games() {
   const { data, refetch } = trpc.match.getAll.useQuery();
   const createMutation = trpc.match.create.useMutation();
-  console.log(data);
 
   return (
     <Layout>
       <div className="@h-full @w-full @max-w-[1200px] @p-5 @grid @grid-rows-[1fr_1fr] @gap-10 allGames">
-        <div className="currentGames">
+        <div id="current-games" className="currentGames">
```
Comment (FemboyDeveloper) [inline src/pages/match/index.tsx:41]: same issue here, make the casing consistent among the class and ID unless there's a specific reason for doing this

**Review by FemboyDeveloper — CHANGES_REQUESTED** [review]
Just a few changes to do, most of the stuff looks good, keep it up dude! Really happy to have someone like you on the team!

File: src/pages/match/index.tsx (hunk 2 above)
Comment (leekahung) [reply src/pages/match/index.tsx:41]: No problem, I can do that. I'll make them consistent with camelCase.

File: src/pages/match/index.tsx (hunk 1 above)
Comment (leekahung) [reply src/pages/match/index.tsx]: Updated to camelCase.

**Review by FemboyDeveloper — APPROVED** [review]
LGTM!

**Pull request merged**